scala/private/phases/phase_write_executable.bzl

```diff
@@ -29,7 +29,6 @@ def phase_write_executable_scalatest(ctx, p):
             "-DRULES_SCALA_MAIN_WS_NAME=%s" % ctx.workspace_name,
             "-DRULES_SCALA_ARGS_FILE=%s" % p.runfiles.args_file.short_path,
         ] + expand_location(ctx, final_jvm_flags),
-        use_jacoco = ctx.configuration.coverage_enabled,
```

Comment:
just so we're clear- you just broke the implicit API of this phase in case someone depends on this to csutomize this behavior of toggling coverage on/off.
I'm ok with it but I think that if we had providers you would have known that (I think you missed it but maybe I'm wrong) and more importantly extenders could easily know that because that provider would have changed and not a nitty gritty in the impl.

Reply:
Isn't this field private to this phase?

Reply:
It seems I was mistaken and it is private